VK_V1.0/VK_V1.0/Friends/FriendsHeaderView.swift

```diff
@@ -3,20 +3,60 @@
 
 import UIKit
 
-/// Компонент кастомного вью
-class FriendsHeaderView: UIView {
-    var view = UIView(frame: CGRect(x: 0, y: 0, width: 400, height: 35))
-
-    override func awakeFromNib() {
-        super.awakeFromNib()
-        view.backgroundColor = .blue
-        addSubview(view)
+/// Кастомный хэдэр
```

Comment:
```suggestion
/// Хэдэр друзей
```
Кастомный слово не используй